microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -13,9 +13,28 @@ keywords: microsoft edge, pwas, pwa, progressive web apps, origin trials
 [!INCLUDE [contact DevTools team note](includes/edge-whats-new-note.md)]
 
 
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 97
+
+Microsoft Edge version 96 is scheduled to become the Stable release on November 18, 2021.  Microsoft Edge version 97 will move from Dev to Beta preview channel following that date.
+
+### Store apps registered as URL handlers hide custom tab UI 
+
+Store-installed PWAs that are participating in the [URL Handler Origin Trial](#url-handlers-origin-trial) will no longer show the Chrome Custom Tab (CCT) when redirecting from the PWA's domain to a matching domain name with a different top-level domain (TLD). This redirecting often happens in apps with locale-specific domains; for example, redirecting from `contoso.com` to `contoso.ca` for users in Canada.
+
+
 <!-- ====================================================================== -->
 ## What's New in Microsoft Edge 96
 
+### URL Protocol Handlers moves to Stable!
+
+The origin trial has been concluded, and "intent to ship" has been signed off.  The URL Protocol Handlers capability is now shipping in Microsoft Edge 96 Stable.
+
+See also:
+*  [Protocol handlers origin trial](#protocol-handlers-origin-trial)
```

Comment:
@Maggers - pls check links for relevance.  Rendered section: https://review.docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa?branch=pr-en-us-1575#url-protocol-handlers-moves-to-stable